tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="NSPDK_candidateClust" name="NSPDK_candidateClusters" version="9.2.3" >
     <requirements>
         <requirement type="package" version="0.6.0">graphclust-wrappers</requirement>
-        <requirement type="package" version="9.2" >nspdk</requirement>
+        <requirement type="package" version="9.2=0" >nspdk</requirement>
```

Comment:
mh, why is that? This is not good? Are the other builds broken?

Reply:
Likely the newer build is broken:
https://travis-ci.org/bgruening/galaxytools/jobs/446966591#L2558